content/hi/reliability.md

```diff
@@ -0,0 +1,7 @@
+---
+title: रिलियबिलिटी (Reliability)
+status: Completed
+category: गुण
+---
+
+क्लाउड नेटिव दृष्टिकोण से, विश्वसनीयता यह दर्शाती है कि सिस्टम विफलताओं के प्रति कितनी अच्छी तरह प्रतिक्रिया करता है। यदि हमारे पास एक वितरित प्रणाली है जो बुनियादी ढांचे में बदलाव के रूप में काम करती रहती है और व्यक्तिगत घटक विफल हो जाते हैं, तो यह विश्वसनीय है। दूसरी ओर, यदि यह आसानी से विफल हो जाता है और इसे चालू रखने के लिए ऑपरेटरों को मैन्युअल रूप से हस्तक्षेप करने की आवश्यकता होती है, तो यह अविश्वसनीय है। क्लाउड नेटिव एप्लिकेशन का लक्ष्य स्वाभाविक रूप से विश्वसनीय सिस्टम बनाना है।
```

Comment:
I think we should maintain consistency by using either `रिलियबिलिटी` or `विश्वसनीयता` everywhere.

Reply:
Sure. I updated with `विश्वसनीयता` @sayantani11